test/validators.js

```diff
@@ -113,6 +113,7 @@ describe('Validators', () => {
         'multiple..dots@gmail.com',
         'wrong()[]",:;<>@@gmail.com',
         '"wrong()[]",:;<>@@gmail.com',
+        '""@invalid.com',
```

Comment:
We can add some tests cases such as 
```
abc-@mail.com,
abc..def@mail.com,
..abc@mail.com,
abc@mail..com,
.abc@mail.com,
abc.def@mail.c,
abc.def@ma@il.c,
abc.def@mail,
abc.def@mail..com,
abc.def@mail#archive.com,
```
For invalid emails, then update our regex to catch these edge cases.